src/averageAges.js

```diff
@@ -55,7 +69,28 @@ function calculateWomenAverageAge(people, withChildren) {
  * @return {number}
  */
 function calculateAverageAgeDiff(people, onlyWithSon) {
-  // write code here
+  const ageDifference = onlyWithSon
+    ? people.filter(child => child.sex === 'm')
+    : people;
```

Comment:
```suggestion
  const ageDifference = onlyWithSon
    ? people.filter(child => child.sex === 'm')
    : people;
```
is it ageDifference?